website/docs/maintenance/configuration.md

```diff
@@ -179,6 +187,10 @@ during the Fluss cluster working.
 | Option                         | Type     | Default | Description                                                                                                        |
 |--------------------------------|----------|---------|--------------------------------------------------------------------------------------------------------------------|
 | kafka.enabled                  | boolean  | false   | Whether enable fluss kafka. Disabled by default. When this option is set to true, the fluss kafka will be enabled. |
-| kafka.port                     | Integer  | 9092    | The port for fluss kafka. The default port is 9092.                                                                |
-| kafka.database                 | String   | _kafka  | The database for fluss kafka. The default database is '_kafka'.                                                    |
-| kafka.connection.max-idle-time | Duration | 60s     | Close kafka idle connections after the given time specified by this config.                                        |
\ No newline at end of file
+| kafka.listener.names           | String   | KAFKA   | The listener names for Kafka wire protocol communication. Support multiple listener names, separated by comma.     |
+| kafka.database                 | String   | _kafka  | The database for fluss kafka. The default database is 'kafka'.                                                     |
+| kafka.connection.max-idle-time | Duration | 60s     | Close kafka idle connections after the given time specified by this config.                                        |
+
+:::warning
+Kafka protocol compatibility is experimental.
```

Comment:
-> `The Kafka protocol compatibility is still in development.`

Move the warning before the kafka config table.